core/src/main/java/com/linecorp/armeria/common/logging/RequestContextExporterBuilder.java

```diff
@@ -40,9 +40,19 @@
     private final ExportGroupBuilder defaultExportGroupBuilder = ExportGroup.builder();
     @Nullable
     private List<ExportGroup> exportGroups;
+    private String name = "DEFAULT";
 
     RequestContextExporterBuilder() {}
 
+    /**
+     * Sets the specified {@code name} for the {@link RequestContextExporter}.
+     */
+    public RequestContextExporterBuilder name(String name) {
+        requireNonNull(name, "name");
+        this.name = name;
+        return this;
+    }
```

Comment:
Is there a case where a user would want to share the same attribute key? If not, we could remove this method and always make sure each `RequestContextExporter` has its own attribute key.